db/migrate/20160720085321_create_container_template.rb

```diff
@@ -0,0 +1,27 @@
+class CreateContainerTemplate < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_templates do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.string     :namespace
```

Comment:
I agree with mooli, container_project is connect to ext_management_system, you can just remove the belongs to ems and place a 
```
  has_one :ext_management_system, through: :container_project

```